e2e/testdata/scheduler/capacity-pod-template.yaml

```
@@ -6,13 +6,13 @@ metadata:
   labels:
     app.kubernetes.io/name: pause
   annotations:
-    capacity.topolvm.io/ssd: %q
+    %sssd: %q
 spec:
   containers:
   - name: pause
     image: registry.k8s.io/pause
     resources:
       requests:
-        topolvm.io/capacity: 1
+        %s/capacity: 1
```

```suggestion
        %s: 1
```

Please use `GetCapacityResource()` in constants.go.

---

Same as https://github.com/topolvm/topolvm/pull/665#discussion_r1123908177

---

Fixed.